core/src/main/java/hudson/util/MultipartFormDataParser.java

```diff
@@ -71,4 +72,31 @@ public void cleanUp() {
         for (FileItem item : byName.values())
             item.delete();
     }
+
+    /**
+     * Checks a <a href="http://www.w3.org/Protocols/rfc1341/4_Content-Type.html">Content-Type</a> string
+     * to assert if it is "multipart/form-data".
+     *
+     * @param contentType <a href="http://www.w3.org/Protocols/rfc1341/4_Content-Type.html">Content-Type</a> string.
```

Comment:
Well this is the wrong RFC :-)  I think you mean rfc1867 which defined multipart/form-data rather than multipart for email....  (and then 1867 is obsoleted by another one which does nothing to define the standard!)

this RFC (1867) also has a mention of a , as a separator for the boundary (ooer missus!) not a comma - so are all the browsers broken out there - I didn't see a tweak of this :-P